wireapp/wire-android
# feat: support material you icons

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Our current blue icon does not follow the guidelines provided by the Design team.

### Solutions

- Follow the design;
- Support Material You launcher icons.

### Dependencies

Because of some res naming conflict, it requires the changes on this Kalium branch, that delete drawables with the same name:

Needs releases with:

- https://github.com/wireapp/kalium/pull/965

### Testing

N/A

#### How to Test

Install the app and see it in your launcher. If you have Material You, select a different colour palette.

### Attachments

Find the Wire app icon on the very bottom. Third icon on top of the Search Bar.

| No Material You Icons | Material You Palette#1 | Material You Palette#2 |
|----------------|------------------|-----------------------|
|![Screenshot_20220923-152325](https://user-images.githubusercontent.com/9389043/191978767-3b314eac-7bf0-4fa3-a436-4924c5da6f8a.png) | ![Screenshot_20220923-152304](https://user-images.githubusercontent.com/9389043/191982188-8bef2016-66a6-45b4-9215-f05628b4dfea.png) | ![Screenshot_20220923-161858](https://user-images.githubusercontent.com/9389043/191982127-69a01361-cd66-443e-bf95-85a74953c094.png)
 |

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
